extensionsdk/Microsoft.Windows.DevHome.SDK/Microsoft.Windows.DevHome.SDK.idl

```diff
@@ -144,7 +144,10 @@ namespace Microsoft.Windows.DevHome.SDK
     [contract(Microsoft.Windows.DevHome.SDK.DevHomeContract, 1)]
     enum AuthenticationState {
         LoggedIn = 0,
-        LoggedOut = 1
+        LoggedOut = 1,
+
+        [contract(Microsoft.Windows.DevHome.SDK.DevHomeContract, 2)] 
```

Comment:
Are there any plans to group up contract version changes or are we not concerned about that right now and it is as needed?